### DISCUSSION TITLE
TextPresenter Caret blink configuration

### DISCUSSION THREAD

**Discussion by sLill** [AvaloniaUI/Avalonia#13665] [Ideas]
It looks like the caret blink interval in TextPresenter is hard-coded to 500 ms. I believe this should be exposed via a StyledProperty on TextPresenter with 0 disabling the blinking behavior. 

**src/Avalonia.Controls/Presenters/TextPresenter.cs**
`  public TextPresenter()
        {
            _caretTimer = new DispatcherTimer { Interval = TimeSpan.FromMilliseconds(500) };
        }
`

**Comment by timunie** [comment]
@sLill the issue was labeled with help-wanted. Are you interested in working on it?

@Gillibald anything against this?

**Reply by Gillibald** [reply]
I am fine with adding such customization

**Reply by sLill** [reply]
@timunie Yes, I've submitted a PR for it. Thank you for taking interest and creating the issue/ticket.
